src/cljs/metafacture_playground/views.cljs

```diff
@@ -278,15 +286,50 @@
         path [:input-fields editor-name]
         collapsed? (re-frame/subscribe [::subs/collapsed? path])
         width (re-frame/subscribe [::subs/editor-width editor-name])]
-    (println editor-name ": width " @width)
     [:> grid-column {:width (or @width (:width config))}
      [:> segment {:raised true}
-      [title-label (:name config)]
-      [collapse-label path]
-      [:> divider {:style {:margin "1.5em 0 0.5em 0"}}]
+      [:> menu
+       {:color color
+        :stackable true}
+       [title-label (:name config)]
+       [collapse-label path]]
       (when-not @collapsed?
         [editor config])]]))
 
+(defn editor-menu-item [config editor]
+  (let [current-editor (re-frame/subscribe [::subs/active-editor])]
+    [:> popup
+     {:content (str (-> editor name clj-str/capitalize)
+                    "-Transformation. Needs a "
+                    (name editor)
+                    "-function in your flux.")
```

Comment:
This looks like a useful message. Where should I see it in the UI? (I see 'WARNING: Flux does not use selected morph/fix').

Reply:
You see this, if you hover over the tab header of morph or fix editor:
![image](https://user-images.githubusercontent.com/72247962/133386047-3bfffdc1-5742-40eb-a93c-987503d58e43.png)

Reply:
:+1: